src/node_kernels/BLTGammaM2015NodeKernel.C

```diff
@@ -139,31 +141,68 @@ BLTGammaM2015NodeKernel::execute(
   sijMag = stk::math::sqrt(2.0 * sijMag);
   vortMag = stk::math::sqrt(2.0 * vortMag);
 
-  TuL = stk::math::min(
-    81.6496580927726 * stk::math::sqrt(tke) / sdr / (minD + 1.0e-10), 100.0);
+  // Computation of the turbulence intensity
```

Comment:
Can we move this comment and the one below to the documentation? And maybe just make a briefer comment here?